### PR TITLE
Use css-modules in web

### PR DESCRIPTION
**Pull request by modrzew**
Instead of:

```jsx
import './Error.css';

class Error extends React.Component {
  render() {
    return <div className="error">error</div>;
  }
}
```

use:

```jsx
import styles from './Error.css';

class Error extends React.Component {
  render() {
    return <div className={styles.error}>error</div>;
  }
}
```

This helps with leaking CSS classes, having you to explicitly apply them to an element.

Two notes:
- all CSS classes should follow `camelCase` pattern instead of `kebab-case` pattern (it's still possible to have `.error-something` in CSS and reference it via `styles.errorSomething`, but probably it's better to just use camelCase everywhere)
- for combining classes, `classnames` package can be used instead of manual string concatenation (if it's not in `package.json` yet, it should be installed with `npm install --save classnames`):

```jsx
import * as classnames from 'classnames';
// several lines later, in render()
<div className={classnames(styles.error, styles.something)}>error</div>
```

### CODE REVIEW

File: web/src/Search.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+html {
```
Comment (modrzew) [inline web/src/Search.css]: @elkmare these can be moved to `index.css` I think

File: web/src/Search.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  width: 480px;
+}
+  
+.SearchHint {
+  padding: 5px 0px 5px 0px;
+  color: #fff;
+  width: 500px;
+}
+  
+.input[type=text] {
```
Comment (modrzew) [inline web/src/Search.css]: @elkmare can you change this to just class? e.g. `.inputText`, and then in the component `className={styles.inputText}`

File: web/src/Search.css (hunk 1 above)
Comment (elkmare) [reply web/src/Search.css]: Done 😄

File: web/src/Search.css (hunk 2 above)
Comment (elkmare) [reply web/src/Search.css]: Done 😄

**Pull request merged**